src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,68 @@
+
+<h3 id="v23-2-5-operational-changes">Operational changes</h3>
+
+- A minimum Raft scheduler concurrency is now enforced per [store]({% link {{ page.version.version }}/cockroach-start.md %}#storage) so that a node with many stores does not spread workers too thinly. This avoids high scheduler latency across replicas on a store when load is imbalanced. [#120798][#120798]
+- A [changefeed]({% link {{ page.version.version }}/change-data-capture-overview.md %}) optimization to reduce duplicates during aggregator restarts has been disabled due to poor performance. [#123596][#123596]
+
+<h3 id="v23-2-5-db-console-changes">DB Console changes</h3>
+
+- The **Commit Latency** chart in the [Changefeed Dashboard]({% link {{ page.version.version }}/ui-cdc-dashboard.md %}) now aggregates by max instead of by sum for multi-node changefeeds. This more accurately reflects the amount of time for events to be acknowledged by the downstream sink. [#121235][#121235]
+
+<h3 id="v23-2-5-bug-fixes">Bug fixes</h3>
+
+- Fixed a slow memory leak when opening many new connections. This bug was introduced in 22.2.9 and 23.1.0. [#121055][#121055]
+- Fixed a bug that occurred when using [`ALTER TABLE`]({% link {{ page.version.version }}/alter-table.md}) to drop and re-add a CHECK constraint with the same name. [#121055][#121055]
+- [Sequence]({% link {{ page.version.version }}/create-sequence.md %}) options `MINVALUE` and `MAXVALUE` automatically adjust to new types bounds. This mirrors the behavior of postgreSQL. [#121309][#121309]
+- Fixed a bug that could prevent timeseries graphs shown on the DB Console SQL Activity [Statement Details]({% link {{ page.version.version }}/statement-details.md %}) page from rendering correctly when specifying a custom time range. [#121383][#121383]
+- Fixed a bug present since at least v21.1 that could lead to incorrect evaluation of an `IN` expression with:
+    - `INT2` or `INT4` type on the left side, and
+    - Values on the right side that are outside of the range of the left side.
+
+    [#121953][#121953]
+- Fixed a leak in reported memory usage (not the actual memory usage) by the internal memory accounting system, the limit for which is configured via the `--max-sql-memory flag` when a long-running sessions issues hundreds of thousands or more transactions. This reporting bug could cause `root: memory budget exceeded` errors for other queries. The bug was introduced in v23.1.17 and f23.2.3. [#121950][#121950]
+- Fixed a bug introduced in v23.2.4 that could prevent collection of table statistics on tables that have virtual computed columns of user-defined type when the newly-introduced [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` is set to `true` (defaults to `false`). The setting is introduced in 23.2.4  and is disabled by default, so only this particular version is affected with non-default setting. [#122319][#122319]
```

Comment:
suggest making 'table statistics', 'computed columns' links to those docs (provided in previous comments)

suggest making cluster settings link a direct anchor link 

version should have a 'v' prepended, e.g. v23.2.4 

suggest 'user-defined type' link to  https://www.cockroachlabs.com/docs/v23.2/create-type